### ISSUE TITLE
Table filter works only without space between words

### ISSUE THREAD

**Issue by dj-fiorex** [wobkenh/simplematcomponents#13]
Hi, i'm here, again ;)
I just noticed a strange behaviour of the table filter, look at the pictures.

This is my table with some fake data
![panoramica](https://user-images.githubusercontent.com/15034741/55798092-5c4fb080-5ace-11e9-9798-41463ffe497e.jpg)

This is the same table when the text on the filter HAS NOT space beetween words
![filtro unica parola](https://user-images.githubusercontent.com/15034741/55798129-7ab5ac00-5ace-11e9-9bee-f96b67082535.jpg)

This is the same table when the text on the filter HAS space beetween words
![filtro con spazio](https://user-images.githubusercontent.com/15034741/55798150-8a34f500-5ace-11e9-9f1e-2e701696adcc.jpg)

Why this happen?? 
I saw your code and it look right:

```typescript
/**
   * Sets DataSource filter using the search string from the search input field.
   *
   * @param filterValue
   */
  applyFilter(filterValue: string) {
    filterValue = filterValue.trim(); // Remove whitespace
    filterValue = filterValue.toLowerCase(); // MatTableDataSource defaults to lowercase matches
    this.dataSource.filter = filterValue;
    if (this.dataSource.filter.length === 0 && this.hasColumnFilter()) {
      this.dataSource.filter = '  ';
    }
  }
```

Please help! Thanks

**Comment by dj-fiorex** [comment]
Thanks for your reply!
I'll watch for this next release!

**Comment by wobkenh** [comment]
This problem should now be fixed with version 2.0 of simplemattable.
